R/observe.R

```diff
@@ -44,7 +44,7 @@ observe_process <- function(linelist) {
 filter_obs_by_obs_time <- function(linelist, obs_time) {
   linelist |>
     mutate(
-      obs_time = obs_time - .data$ptime,
+      relative_obs_time = obs_time - .data$ptime,
       censored_obs_time = .data$obs_time - .data$ptime_lwr,
```

Comment:
should this be `obs_time` instead of `.data$obs_time`? More so in line 51 - otherwise not filtering based on the `obs_time` argument

Reply:
Yes this does look very suspicious, thanks for noting. For some reason I was just focsed on the downstream and wasn't really thinking about the function itself..